dev/InfoBar/TestUI/InfoBar.cs

```diff
@@ -79,16 +84,22 @@ public InfoBar()
 
         protected override void OnApplyTemplate()
         {
-            _alternateCloseButton = GetTemplateChild<Button>("AlternateCloseButton");
+            _actionButton = GetTemplateChild<Button>("ActionButton");
+            _title = GetTemplateChild<TextBlock>("Title");
+            _message = GetTemplateChild<TextBlock>("Message");
+            _hyperlinkButton = GetTemplateChild<HyperlinkButton>("HyperlinkButton");
+            _standardIcon = GetTemplateChild<IconSourceElement>("StandardIcon");
+            _userIcon = GetTemplateChild<IconSourceElement>("UserIcon");
+            _contentRootGrid = GetTemplateChild<Grid>("ContentRootGrid");
             _closeButton = GetTemplateChild<Button>("CloseButton");
             _actionButton = GetTemplateChild<Button>("ActionButton");
             _myContainer = GetTemplateChild<Border>("Container");
 
             UpdateButtonsState();
             UpdateSeverityState();
             OnIsOpenChanged();
+            UpdateMargins();
 
-            _alternateCloseButton.Click += new RoutedEventHandler(OnCloseButtonClick);
             _closeButton.Click += new RoutedEventHandler(OnCloseButtonClick);
```

Comment:
>_closeButton [](start = 12, length = 12)

You'll need to make sure that this is not null incase someone has retemplated your control and removed the close button

Reply:
That makes a lot of sense. Fixed with the most recent push.